packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -222,6 +222,19 @@ const start = async (zcf, privateArgs) => {
     zcf.reallocate(offerToSeat, collateralSeat);
   };
 
+  const burnRUNToReduceShortfall = reduction => {
```

Comment:
RUN is all-caps even in function names?

Reply:
That was the policy I understood. I wouldn't mind if that's not the rule.

Reply:
can wait for https://github.com/Agoric/agoric-sdk/issues/4800